common/types.h

```diff
@@ -12,6 +12,7 @@ namespace slog {
 using Key = std::string;
 using Value = std::string;
 using TxnId = uint32_t;
+using TxnReplicaId = uint32_t;
```

Comment:
So far I'm not assigning to this, I could use your help figuring out the way to approach it. I think that can go in the following PR for the scheduler though